contrib/opentimelineio_contrib/adapters/tests/test_rvsession.py

```diff
@@ -468,14 +468,20 @@
     "RV Adapter does not work in python 3."
 )
 class RVSessionAdapterReadTest(unittest.TestCase):
+    def setUp(self):
+        self.fd, self.tmp_path = tempfile.mkstemp(suffix=".rv", text=True)
+
+    def tearDown(self):
+        os.close(self.fd)
+        os.unlink(self.tmp_path)
+
     def test_basic_rvsession_read(self):
         timeline = otio.adapters.read_from_file(SCREENING_EXAMPLE_PATH)
-        tmp_path = tempfile.mkstemp(suffix=".rv", text=True)[1]
 
-        otio.adapters.write_to_file(timeline, tmp_path)
-        self.assertTrue(os.path.exists(tmp_path))
+        otio.adapters.write_to_file(timeline, self.tmp_path)
+        self.assertTrue(os.path.exists(self.tmp_path))
```

Comment:
You can omit this assertion because the `open` on the next line will exercise that for you.